src/screens/OrganizationDashboard/OrganizationDashboardMocks.ts

```diff
@@ -68,59 +67,112 @@ export const MOCKS = [
-              lastName: 'Doe',
-              email: 'johndoe@gmail.com',
+        postsByOrganizationConnection: {
+          edges: [
+            {
+              _id: '6411e53835d7ba2344a78e21',
+              title: 'Post 15',
+              text: 'This is the first post that was made',
+              imageUrl: null,
+              videoUrl: null,
+              creator: {
+                _id: '640d98d9eb6a743d75341067',
+                firstName: 'Aditya',
+                lastName: 'Shelke',
+                email: 'adidacreator1@gmail.com',
+              },
+              createdAt: dayjs(new Date()).add(1, 'day'),
+              likeCount: 0,
+              commentCount: 0,
+              comments: [],
+              likedBy: [],
             },
-          },
-        ],
+            {
+              _id: '6411e54835d7ba2344a78e29',
+              title: 'Post 2',
+              text: 'Hey, anyone saw my watch that I left at the office?',
+              imageUrl: null,
+              videoUrl: null,
+              creator: {
+                _id: '640d98d9eb6a743d75341067',
+                firstName: 'Aditya',
+                lastName: 'Shelke',
+                email: 'adidacreator1@gmail.com',
+              },
+              createdAt: dayjs(new Date()).add(1, 'day'),
+              likeCount: 0,
+              commentCount: 2,
+              comments: [
+                {
+                  _id: '64eb13beca85de60ebe0ed0e',
+                  creator: {
+                    _id: '63d6064458fce20ee25c3bf7',
+                    firstName: 'Noble',
+                    lastName: 'Mittal',
+                    email: 'test@gmail.com',
+                    __typename: 'User',
+                  },
+                  likeCount: 1,
+                  likedBy: [
+                    {
+                      _id: 1,
+                    },
+                  ],
+                  text: 'Yes, that is $50',
+                  __typename: 'Comment',
+                },
+                {
+                  _id: '64eb483aca85de60ebe0ef99',
+                  creator: {
+                    _id: '63d6064458fce20ee25c3bf7',
+                    firstName: 'Noble',
+                    lastName: 'Mittal',
+                    email: 'test@gmail.com',
+                    __typename: 'User',
+                  },
+                  likeCount: 0,
+                  likedBy: [],
+                  text: 'Great View',
+                  __typename: 'Comment',
+                },
+              ],
+              likedBy: [
+                {
+                  _id: '63d6064458fce20ee25c3bf7',
+                  firstName: 'Comment',
+                  lastName: 'Likkert',
+                },
+              ],
+            },
+          ],
+        },
       },
     },
   },
   {
     request: {
-      query: ORGANIZATION_EVENT_LIST,
+      query: ORGANIZATION_EVENT_CONNECTION_LIST,
+      variables: {
+        organization_id: '123',
```

Comment:
@JamarTG you have used `organization_id: '123'` here in the mocks. Have you assigned the organization_id as `123` in the tests?